testing/templates/allow-whitespaces.html

```diff
@@ -35,7 +35,7 @@
 {{1+2}}{{ 1+2 }}{{ 1 +2 }}{{ 1+ 2 }} {{ 1 + 2 }}
 {{1*2}}{{ 1*2 }}{{ 1 *2 }}{{ 1* 2 }} {{ 1 * 2 }}
 {{1&2}}{{ 1&2 }}{{ 1 &2 }}{{ 1& 2 }} {{ 1 & 2 }}
-{{1|2}}{{ 1|2 }}{{ 1 |2 }}{{ 1| 2 }} {{ 1 | 2 }}
+{{1 bitor 2}}{{ 1 bitor 2 }}{{ 1 bitor 2}}{{1 bitor 2 }} {{1			bitor			2}}
```

Comment:
Why this weird tab change? Wouldn't adding whitespace characters before `1` and after `2` provide the expected result?

Reply:
I had no idea what to do in the last test. :) If I remove it, [testing/tests/whitespace.rs:42](https://github.com/rinja-rs/rinja/pull/18/files#diff-b63937165590fc2719d8f31a1554a4e4b98c56f5dece3bd96b73fe7827a3f629) changes.

Reply:
Oh well, let's keep it as is for now, I'll try to take a look later on.